### PR TITLE
re-arrange readme to put most important info for users first

### PR DESCRIPTION
**Pull request by fangohr**

### CODE REVIEW

**Comment by iamashwin99** [comment]
Is this MR ready for review?

**Comment by fangohr** [comment]
> Is this MR ready for review?

Not yet.

**Comment by fangohr** [comment]
Ready for review now @iamashwin99 .

Main purpose is to explain in the first section how to use this, with minimal detail beyond that. Experts and developers can continue reading.

File: README.rst [hunk 1]
```diff
@@ -1,21 +1,95 @@
+Repository for building and executing the `OCTOPUS code
+<http://octopus-code.org>`__ package in a Docker container. 
+
+Use case: run Octopus (for small calculations and tutorials) conveniently in
```
Comment (iamashwin99) [inline README.rst:4]:
```suggestion
Use case: run Octopus (for small calculations and tutorials) conveniently in a
```

File: README.rst [hunk 2]
```diff
@@ -1,21 +1,95 @@
...
+    docker run --rm -ti -v $PWD:/io fangohr/octopus octopus
+  
+   The first time you run this, Docker needs to download the image
+   ``fangohr/octopus`` from DockerHub. This could take a while (depending on your
+   internet connection, the image size is about 850MB).
+
+   Meaning of the switches:
+   
+   - ``--rm`` remove docker container after it has been carried out (good practice to reduce disk usage).
+   - ``-ti`` allow to make the Terminal session Interactive
```
Comment (iamashwin99) [inline README.rst:36]: perhaps explain the use of both ``t`` and ``i`` flags (see https://docs.docker.com/engine/reference/run/#commands-and-arguments)
```suggestion
   - ``-ti`` start an ``i``nteractive  pseudo-``t``ty shell in the container 
```

File: README.rst [hunk 3]
```diff
@@ -1,21 +1,95 @@
...
+  Only the ``octopus`` command will be carried out in the
+  container. Any output files are written to the current directory on the host.
+
+- carry out data analysis on the host
+
+If you want to work interactively *inside* the container, replace the name of the executable with ``bash``::
+
+  docker run --rm -ti -v $PWD:/io fangohr/octopus bash
+  
+You are then the root user in the container. Octopus was compiled in ``/opt/octopus*``. There are also some trivial examples in ``/opt/octopus-examples``.
```
Comment (iamashwin99) [inline README.rst:81]: expanding on what the examples are
```suggestion
You are then the root user in the container. Octopus was compiled in ``/opt/octopus*``. There are also some trivial example input files in ``/opt/octopus-examples``.
```

File: README.rst [hunk 4]
```diff
@@ -33,17 +107,17 @@ inside the container.
...
 - Step 2: use the container to execute Octopus inside the container
 
 
 Step 1: How obtain a Docker container image with Octopus
 --------------------------------------------------------
 
-We provide a `Dockerfile <Dockerfile>`__ to compile Octopus 13.0
-and `Dockerfile-develop <Dockerfile-develop>`__ to compile the ``develop`` branch of the Octopus
-repository in a container.
+In this repository we provide a `Dockerfile <Dockerfile>`__ to compile Octopus
```
Comment (iamashwin99) [inline README.rst:118]:
```suggestion
In this repository, we provide a `Dockerfile <Dockerfile>`__ to compile Octopus
```

File: README.rst [hunk 5]
```diff
@@ -73,55 +147,25 @@ Instead of building it yourself, you can also pull an image from Dockerhub
...
-  Only the ``octopus`` command will be carried out in the
-  container. Any output files are written to the current directory on the host.
-- carry out data analysis on the host
-
-If you want to work interactively *inside* the container, replace the name of the executable with ``bash``::
-
-  docker run --rm -ti -v $PWD:/io octimage bash
-  
-You are then the root user in the container. Octopus was compiled in ``/opt/octopus*``. There are also some trivial examples in ``/opt/octopus-examples``.
+See Quick start section above for more details.
```
Comment (iamashwin99) [inline README.rst:162]: Is it possible to somehow put a cross reference in rst? we could then add a hyperlink to that section. I did do a brief search but didn't find a straight forward solution.

File: README.rst [hunk 6]
```diff
@@ -135,3 +179,12 @@ or use ``uname -m`` inside the container.)
    :target: https://github.com/fangohr/octopus-in-docker/actions/workflows/debian-develop.yml
 
 
+Status
```
Comment (iamashwin99) [inline README.rst:182]: one suggestion might be to move this to the top, that way you can see the pipeline status without scrolling. Either way is fine.

**Review by iamashwin99 — APPROVED** [review]
Looks good!,
few minor suggestions / questions.

File: README.rst (hunk 5 above)
Comment (fangohr) [reply README.rst:162]: I also didn't find a solution quickly. The problem is that GitHub does not support all rst features. (There is a solution in general, but it didn't work on GitHub when I tried.)

File: README.rst [hunk 7]
```diff
@@ -135,3 +179,12 @@ or use ``uname -m`` inside the container.)
...
+======
+
+Status of building the Docker images:
+
+|stable| Debian Bookworm (12), Last octopus release (13.0)
```
Comment (iamashwin99) [inline README.rst:187]:
```suggestion
|stable| Debian Bookworm (12), Latest octopus release (13.0)
```

File: README.rst (hunk 6 above)
Comment (fangohr) [reply README.rst:182]: We had it at the top in the beginning, but I thought it is of little use to the users of the container.

**Pull request merged**